source/index.js

```diff
@@ -78,9 +78,8 @@ function generateAdsTxt(manifest, header, footer) {
 }
 
 function generateLineForField(field) {
-    const domainExp = getDomainRegex();
     const { domain, publisherAccountID, accountType, certificateAuthorityID, comment } = field;
-    if (domainExp.test(domain) !== true) {
+    if (!isDomainName(domain, false)) {
```

Comment:
Ok, I thought we were going to use is-valid-domain instead?

If you keep `is-domain-name`, the `false` as the second parameter isn't required, as it's the default value.